setup.py

```diff
@@ -98,11 +107,11 @@ def write_version_py(filename):
     elif os.path.exists('kiva/_version.py'):
         # must be a source distribution, use existing version file
         try:
-            from kiva._version import git_revision, full_version
-        except ImportError:
-            raise ImportError("Unable to import git_revision. Try removing "
-                              "kiva/_version.py and the build directory "
-                              "before building.")
+            git_revision, full_version = read_version_py('kiva/_version.py')
```

Comment:
Probably need to be careful here:

* cross-platform path construction
* need to ensure that it works if `setup.py` is not in current working dir (ie. base off of `os.dirname(__file__)`).